docs/tutorials/colorspace.ipynb

```diff
@@ -0,0 +1,420 @@
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/io/blob/master/docs/tutorials/colorspace.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/io/blob/master/docs/tutorials/colorspace.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "      <td>\n",
+        "    <a href=\"https://raw.githubusercontent.com/tensorflow/io/master/docs/tutorials/colorspace.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "xHxb-dlhMIzW"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "In computer vision, the selected color space could have a significant the performance of the model. While `RGB` is the most common color space, in manay situations the model performs better when switching to alternative color spaces such as `YUV`, `YCbCr`, `XYZ (CIE)`, etc. \n",
+        "\n",
+        "The `tensorflow-io` package provides a list of color space conversions APIs that can be used to prepare and augment the image data."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install required Packages, and restart runtime"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uUDYyMZRfkX4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!pip install tensorflow-io-nightly"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VSkY6UAxODOq",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Download the sample image\n",
+        "\n",
+        "The image example used in this tutorial is from a downloadable screenshot of [Fallout 4](https://fallout.bethesda.net/en/games/fallout-4), available in the media section of the website. For convenience, a [copy](https://raw.githubusercontent.com/tensorflow/io/master/tests/test_image/fallout-4.jpg) has been saved on GitHub. The image sample could be replaced by any JPEG image for this tutorial.\n",
```

Comment:
We do end up republishing the images so we always want to be careful about image copyright,. and it's best if we pull images from resources that we actually control.

So could you switch this to something from download.tensorflow.org/example_images? Any of these would work:

https://storage.googleapis.com/download.tensorflow.org/example_images/
* grace_hopper.jpg.
* 320px-Felis_catus-cat_on_snow.jpg
* Green_Sea_Turtle_grazing_seagrass.jpg
* San_Francisco_fireboat_showing_off.jpg
*  YellowLabradorLooking_new.jpg